tests/network/l2_bridge/nad_ref_change/test_nad_ref_change.py

```
@@ -52,10 +66,13 @@ class TestRunningVMLinuxBridgeVlanChange:
         - No TCP connectivity between the under-test VM and the reference VM on NAD-VLAN-B
     """
 
-    __test__ = False
-
     @pytest.mark.polarion("CNV-15945")
-    def test_vm_state_iface_info_preserved(self):
+    def test_vm_state_iface_info_preserved(
+        self,
+        under_test_vm_two_ifaces,
+        bridge_nad_a,
```

Ruff (0.15.13)

[warning] Unused method argument: `bridge_nad_a`

(ARG002)